Mirror/Runtime/NetworkTransform.cs

```diff
@@ -1232,8 +1234,9 @@ static float ReadAngle(NetworkReader reader, CompressionSyncMode compression)
                 case CompressionSyncMode.None:
                     return reader.ReadSingle();
                 case CompressionSyncMode.Low:
+                    return reader.ReadInt16() * (2 * Math.PI / short.MaxValue);
```

Comment:
I could be wrong,  but does this even compile?   don't you need to cast the whole thing to float?

Reply:
Something went sideways. Ill try again